wire/msgmixpairreq.go

```diff
@@ -0,0 +1,426 @@
+// Copyright (c) 2023 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"bytes"
+	"fmt"
+	"hash"
+	"io"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+const (
+	// MaxMixPairReqScriptClassLen is the maximum length allowable for a
+	// MsgMixPairReq script class.
+	MaxMixPairReqScriptClassLen = 32
+
+	// MaxMixPairReqUTXOs is the maximum number of unspent transaction
+	// outputs that may be contributed in a single mixpairreq message.
+	MaxMixPairReqUTXOs = 512 // XXX: PNOOMA
```

Comment:
Reminder about this.  Similar to the other cases, even though they might be PNOOMA, I'm sure there was a least some justification you used to arrive at these figures.

Rather than leaving an XXX, please document the rationale for the number even if it's something as simple as being based on a empirical data gathered from running the centralized mixing service plus some extra for additional wiggle room.